docs/examples/Document.lbf

```diff
@@ -35,14 +38,5 @@ prod RichDocument = (Document RichContent)
 -- ## We need a list type
 sum List a = Nil | Cons a (List a)
 
```

Comment:
If this is a Prelude documentation, then we can use Prelude.List opaque as well?